cores/esp8266/umm_malloc/umm_malloc.cpp

```diff
@@ -498,9 +498,30 @@
 #include "umm_malloc.h"
 
 #include "umm_malloc_cfg.h"   /* user-dependent */
+#if defined(DEBUG_ESP_PORT)
+#include <assert.h>
+#endif
 
 extern "C" {
 
+#ifdef UMM_CRITICAL_PERIOD_ANALYZE
+static struct _UMM_TIME_STATS time_stats = {
```

Comment:
Suggest moving this static global to another .cpp, as previously explained.

Reply:
okay